### ISSUE TITLE
Nan while joint training (descriptor + detector)

### ISSUE THREAD

**Issue by jeethesh-pai** [eric-yyjau/pytorch-superpoint#65]
Hi Eric,

I was training for descriptor + detector after two homographic adaptations for detector. But after some epochs, I get nan loss. When I see traceback it says that the convolution layer produced nan values. Do you have any ideas about what is causing this problem? Is it the descriptor loss that is becoming too less as training continues?

Thanks a lot for your reply
